pkg/errinsrc/errinsrc.go

```diff
@@ -88,16 +88,21 @@ func (e *ErrInSrc) Bailout() {
 	panic(Bailout{List: List{e}})
 }
 
+func (e *ErrInSrc) Title() string {
+	return e.Params.Title
+}
+
 func (e *ErrInSrc) Error() string {
 	var b strings.Builder
 
 	// Write the header
 	const headerGrayLevel = 12
 	const spacing = 4 + 2 + 7 // (4 = "--" on both sides, 2 = " " on the sides of the title, 7 = "[E0000]")
+	b.WriteRune('\n')         // Always start with a new line as these errors are expected to be full screen
 	b.WriteString(aurora.Gray(headerGrayLevel, fmt.Sprintf("%c%c ", set.HorizontalBar, set.HorizontalBar)).String())
 	b.WriteString(aurora.Red(e.Params.Title).String())
 	b.WriteByte(' ')
-	b.WriteString(aurora.Gray(headerGrayLevel, strings.Repeat(string(set.HorizontalBar), errorWidth-len(e.Params.Title)-spacing)).String())
+	b.WriteString(aurora.Gray(headerGrayLevel, strings.Repeat(string(set.HorizontalBar), ErrorWidth-len(e.Params.Title)-spacing)).String())
```

Comment:
With very small terminals these can become negative. Do we handle that? Should we enforce a minimum value?